templates/unbound.conf.erb

```diff
@@ -10,6 +10,13 @@ server:
   use-syslog: no
 <% else -%>
   use-syslog: yes
+<% if @do_not_query_address.is_a? Array -%>
```

Comment:
We've set the data type of `do_not_query_address` to be an array, so we probably don't need to check the data type.